lib/apiservers/portlayer/swagger.yml

```diff
@@ -249,7 +249,7 @@ paths:
             $ref: "#/definitions/VolumeResponse"
         '500':
           description: "Error"
-          $schema:
+          schema:
```

Comment:
this error should be noted as `swagger validate` does not catch it however it does generate code still. Because of this it can actually affect expected response codes.